### PR TITLE
Updated URL ref parameter values for analytics for PhotoDetails.vue

### LINKED ISSUE #181: Update URL `ref` parameter values for analytics
Currently we add `ref=ccsearch` to track the link clicks:

https://github.com/WordPress/openverse-frontend/blob/db693f08f2046c24808addf9f2636352d010e062/src/components/ImageDetails/PhotoDetails.vue#L242

https://github.com/WordPress/openverse-frontend/blob/7668b19af296a77297ab9516acbf4312ed473e73/src/components/Filters/LicenseExplanationTooltip.vue#L50-L54

We need to update them with `openverse`, where necessary.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by MuhammadFaizanHaidar**
Replace ccsearch with openverse
issue track: https://github.com/WordPress/openverse-frontend/issues/181

## Fixes

Fixes #181 by @obulat

## Description
Replaces ref=ccsearch with ref=openverse
## Screenshots
ccsearchtoopenverse
## Technical details
Updated URL ref parameter values for analytics.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by MuhammadFaizanHaidar** [comment]
Can someone please tell why PR label checks are getting failed. Is this because I made two PR's to some issue? Although their titles are different. Thanks

File: src/components/ImageDetails/PhotoDetails.vue [hunk 1]
```diff
@@ -236,7 +236,7 @@ export default {
         : ''
     },
     ccLicenseURL() {
```
Comment (obulat) [inline src/components/ImageDetails/PhotoDetails.vue:238]:
```suggestion
    openverseLicenseURL() {
```

Comment (obulat) [reply src/components/ImageDetails/PhotoDetails.vue:238]: Could you also please change the prop name from `ccLicenseURL` to `openverseURL`? On lines 130, 145, and inside the corresponding components.

**Review by obulat — CHANGES_REQUESTED** [review]

**Comment by obulat** [comment]
> Can someone please tell why PR label checks are getting failed. Is this because I made two PR's to some issue? Although their titles are different. Thanks

To simplify creating the changelog, we have a script in the repository that checks if you have applied relevant tags to your PR.  At least one of the `aspect:<>` and one of the `goal:<>` labels need to be applied. I've added the labels, so the checks pass now.

File: src/components/ImageDetails/PhotoDetails.vue (hunk 1 above)
Comment (MuhammadFaizanHaidar) [reply src/components/ImageDetails/PhotoDetails.vue:238]: yes sure openverseURL or openverseLicenseURL? @obulat

**Comment by MuhammadFaizanHaidar** [comment]
> Can someone please tell why PR label checks are getting failed. Is this because I made two PR's to some issue? Although their titles are different. Thanks

@obulat I can see 2 other files   using ccLicenseUrl as a prop.
1. ImageAttributes.vue
2. ImageInfo.vue

**Review by obulat — APPROVED** [review]

**Comment by obulat** [comment]
This changes should be enough to ensure that the `ref` parameter is correct. We will change the prop names in future PRs. Thank you for your contribution, @MuhammadFaizanHaidar !

**Comment by MuhammadFaizanHaidar** [comment]
> This changes should be enough to ensure that the `ref` parameter is correct. We will change the prop names in future PRs. Thank you for your contribution, @MuhammadFaizanHaidar !

@obulat I am happy that I got a chance to give back to my community!

**Pull request merged**